### PR TITLE
Olivia components

### PR DESCRIPTION
**Pull request by oliviaclyde**
Removed duplicate code in form bank and form list, completed top section, court case info and sections 1-3 of web form component file

### CODE REVIEW

File: frontend/app/forms/bank/form-bank.component.js [hunk 1]
```diff
@@ -231,16 +234,6 @@ export default class FormBank extends React.Component {
             />
           )}
         />
-        <Route
```
Comment (joeldenning) [inline frontend/app/forms/bank/form-bank.component.js]: This was a duplicate of line 208? Good catch

File: frontend/app/forms/bank/form-list.component.js [hunk 2]
```diff
@@ -301,22 +301,6 @@ export default class FormList extends React.Component {
                 "/static/forms/petition-for-expungement-from-bci/08_Order-special_certificate-3.png"
               ]}
             />
-            <FillableForm
```
Comment (joeldenning) [inline frontend/app/forms/bank/form-list.component.js]: Duplicate of 276? nice catch

File: frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js [hunk 3]
```diff
@@ -1,5 +1,186 @@
...
+      <Section name="Personal Information">
+        {/* <TextInput dataKey="person.firstName" label={__("First Name")} />
+        <TextInput dataKey="person.middleName" label={_("Middle Name")} />
+        <TextInput dataKey="person.lastName" label={_("Last Name")} />
+        <TextInput dataKey="person.addressStreet" label={_("Street Address")} />
+        <TextInput dataKey="person.addressCity" label={_("City")} />
+        <TextInput dataKey="person.addressState" label={_("State")} />
+        <TextInput dataKey="person.addressZip" label={_("Zip Code")} />
+        <TextInput dataKey="person.phone" label={_("Phone")} />
+        <TextInput dataKey="person.email" label={_("Email")} /> */}
```
Comment (joeldenning) [inline frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js]: Why are these commented lines of code here? They look the same as the next lines of code - let's delete these commented lines.

File: frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js [hunk 4]
```diff
@@ -1,5 +1,186 @@
...
+        />
+      </Section>
+
+      <Section name="1. Documents Served">
+        <small className="web-form-input">
+          The following documents were served by the method described below
+        </small>
+        <small className="web-form-input">(Choose all that apply:)</small>
+        <Checkbox
+          dataKey="proofOfCompletedService_case.summons"
```
Comment (joeldenning) [inline frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js]: In data keys, the part before the period represents the Firebase collection for the data key. We currently only have two firebase collections - `case` and `person`. We should not introduce new ones without careful consideration. See [this related comment](https://github.com/JustUtahCoders/utahexpungements.org/pull/109#discussion_r341833909) which explains this more.

```suggestion
          dataKey="case.proofOfServiceSummons"
```

File: frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js [hunk 5]
```diff
@@ -1,5 +1,186 @@
...
+        <Checkbox
+          dataKey="proofOfCompletedService_case.parentingPlan"
+          label="Parenting Plan"
+        />
+
+        <Checkbox
+          dataKey="proofOfCompletedService_case.other"
+          label="Other (describe)"
+        />
+        {/* {case.proofOfCompletedService_case && (
```
Comment (joeldenning) [inline frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js]: Do we need this? Or can it be removed?

File: frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js [hunk 6]
```diff
@@ -1,5 +1,186 @@
...
+        <p className="web-form-input">
+          I have not been convicted of a felony violation of a sex offsense{" "}
+          <small>(Listed in Utah Code 77-41-102(16)</small>.
+        </p>
+        <p className="web-form-input">
+          I am not a respondent in a protective order proceeding
+          <small>(Utah Code 78B-7-101 et seq.)</small>.
+        </p>
+      </Section>
+      <Section name="4. Service by Third Person (continued)"></Section>
```
Comment (joeldenning) [inline frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js]: Are you planning on doing this section in a follow-up PR?

File: frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js (hunk 3 above)
Comment (oliviaclyde) [reply frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js]: Fixed translation labels and removed duplicated commented out code

File: frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js (hunk 4 above)
Comment (oliviaclyde) [reply frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js]: Switched from proofOfCompletedService_case to case

File: frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js (hunk 6 above)
Comment (oliviaclyde) [reply frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js]: Yes. In a new PR

File: frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js (hunk 5 above)
Comment (oliviaclyde) [reply frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js]: Removed and now describe field appears only if Other is checked

**Pull request merged**